### PR TITLE
Sunset grml32

### PR DESCRIPTION
**Pull request by zeha**

### CODE REVIEW

File: content/post/416-grml32-sunset.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+In the meantime the PC landscape changed a lot. [Intel jumped](https://en.wikipedia.org/wiki/X86-64) on the "amd64" train, and abandoned their own ia64 ("Itanium") design. Modern PCs often do not properly support the old 32-bit architecture anymore. Even PCs manufactured ten years ago can boot grml32 and grml64. No new 32-bit only CPUs are manufactured, and no new features got added. Testing grml32 became a lot harder, and testing on a "real 32-bit only" hardware is now impossible for us.
+
+Linux kernel upstream also is losing interest and time to support the 32-bit architecture (just called "x86") there. The Debian kernel team decided to [stop building Linux kernel packages](https://salsa.debian.org/kernel-team/linux/-/commit/a18e683900b4f5ffb30dd18b8481cbdc66e7a34a) starting with Linux 6.11.
+
+As Grml releases are dependent on Debian kernel packages, this decision also *ends the line* for grml32.
+
+
```
Comment (jkirk) [inline content/post/416-grml32-sunset.md]: nit: seems like an unneeded new line.

File: content/post/416-grml32-sunset.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+Users really needing grml32 due to lack of 64-bit support of their hardware can continue to use Grml 2024.02, which will be the last release with a grml32 build. However, we encourage users to migrate to a more modern, possibly second-hand, hardware instead.
```
Comment (jkirk) [inline content/post/416-grml32-sunset.md]: Dropped this line in deepL. It gave me:

> Users who really need grml32 due to lack of 64-bit support on their hardware can still use Grml 2024.02, which will be the last release with a grml32 build. However, we encourage users to migrate to more modern, possibly used, hardware instead.

Maybe better?

**Review by jkirk — COMMENTED** [review]
Apart of the two things. LGTM! Nice text!

File: content/post/416-grml32-sunset.md (hunk 1 above)
Comment (zeha) [reply content/post/416-grml32-sunset.md]: fixed

File: content/post/416-grml32-sunset.md (hunk 2 above)
Comment (zeha) [reply content/post/416-grml32-sunset.md]: force-pushed

**Review by mika — APPROVED** [review]
Well written, thanks!

**Pull request merged**